iotlabcli/integration/test_cli_tools_help.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
Talking about the parser tests, these are "dev unit-tests", hitting on internal code, not "integration tests", hitting on user-visible features.  Sure they both "sort of" test the same thing ; I see them as complimentary.  Are we bickering for 30 sloc ? No big deal, right ?

I might be biased, but I find the pytest-based test to be neat, simple, all in a single place, for all tools ; and it shows all available cli interface tools and commands in a snap.